hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/FSNamesystem.java

```diff
@@ -8308,25 +8255,24 @@ public Object performTask() throws IOException {
     return (PathInformation)handler.handle(this);
   }
 
-  public static int dbOnDiskFileMaximumSize() {
-    return DB_ON_DISK_FILE_MAX_SIZE;
+  public static int getMaxSmallFileSize() {
+    return DB_MAX_SMALL_FILE_SIZE;
   }
 
-
-  public static int dbOnDiskSmallFileMaxSize() {
-    return DB_ON_DISK_SMALL_FILE_MAX_SIZE;
+  public static int getDBOnDiskSmallBucketSize() {
+    return DB_ON_DISK_SMALL_BUCKET_SIZE;
   }
 
-  public static int dbOnDiskMediumFileMaxSize() {
-    return DB_ON_DISK_MEDIUM_FILE_MAX_SIZE;
+  public static int getDBOnDiskMidumBucketSize() {
```

Comment:
typo: getDBOnDiskMediumBucketSize instead of getDBOnDiskMidumBucketSize

Reply:
fixed typo